sdk/data/azappconfig/client.go

```diff
@@ -141,7 +141,9 @@ func (c *Client) GetSetting(ctx context.Context, key string, options *GetSetting
 
 	var lastModified *time.Time
 	if resp.LastModified != nil {
-		tt, err := time.Parse(http.TimeFormat, *resp.LastModified)
+		slm := *resp.LastModified
+		tflm := slm.Format(http.TimeFormat)
+		tt, err := time.Parse(http.TimeFormat, tflm)
```

Comment:
I think these changes should be reverted?

Reply:
Yes we can revert this. It is displaying as an error locally with a failure to convert the `*resp.LastModified` (type time.time) to string. I reran it and this doesn't change behavior

Reply:
We do still need this change. Build is failing:
![image](https://github.com/Azure/azure-sdk-for-go/assets/89044647/7da26ed0-8b47-40ab-aa65-1f129ee5b34b)

Reply:
Looking closer, I see that the `LastModified` field in type `AzureAppConfigurationClientGetKeyValueResponse` has been removed. It appears to overlap with a field with the same name in `KeyValue` which is already in `time.Time` format. So we shouldn't need to perform any parsing/conversion.